controllers/admin/AdminModulesController.php

```diff
@@ -660,14 +629,7 @@ public function postProcessCallback()
                     $modules = (array) $modules;
                 }
             } elseif ($key == 'updateAll') {
-                $loggedOnAddons = false;
-
-                if (!empty($this->context->cookie->username_addons)
-                    && !empty($this->context->cookie->password_addons)) {
-                    $loggedOnAddons = true;
-                }
-
-                $allModules = Module::getModulesOnDisk(true, $loggedOnAddons, $this->context->employee->id);
+                $allModules = Module::getModulesOnDisk(true, false, $this->context->employee->id);
```

Comment:
Even though it would be a BC Break, we can maybe delete the second argument of `Module::getModulesOnDisk` as it won't be used anymore?

Reply:
Right, I do it.